src/openrct2/world/TileElement.cpp

```diff
@@ -18,8 +18,10 @@
 #include "Banner.h"
```

Comment:
You can probably remove this include, I was wondering why ::GetBanner compiles in Banner.cpp which is explained by this header having it. Might have to adjust a few other files as well.